src/modules/ROOT/pages/changelog/202303/02.json

```diff
@@ -182,15 +187,17 @@
     },
     {
       "what": [
-        "missing sessionID parameter in automation session search. The sessionID value wasn't taken into consideration when filtering an automation session list."
+        "missing sessionID parameter in automation session search.",
+        "The sessionID value wasn't taken into consideration when filtering an automation session list."
       ],
       "refs": [
         "https://github.com/cortezaproject/corteza/commit/37035984aed08d1c30c90c59a8aeb9d6d2449e31"
       ]
     },
     {
       "what": [
-        "the value of the SMTP_FROM variable in the server .env variables. The variable wasn't taken into consideration is a value was added to it."
+        "the value of the `SMTP_FROM` variable in the server `.env` variables.",
```

Comment:
This is a change, not a fix.
Initially, we only had the .env option which we then changed to a configurable thing in settings and we're not again changing to prioritize .env options -- changing expected behaviour